src/porepy/models/incompressible_flow_model.py

```diff
@@ -224,6 +237,23 @@ def _assign_variables(self) -> None:
                     self.mortar_variable: {"cells": 1},
                 }
 
+    def _create_dof_and_eq_manager(self) -> None:
+        self.dof_manager = pp.DofManager(self.gb)
+        self._eq_manager = pp.ad.EquationManager(self.gb, self.dof_manager)
+
+    def _create_assembler(self) -> None:
```

Comment:
If we only use Ad, we don't need the `Assembler`.

Reply:
Incompressible_flow_model.py supports the case without AD. For instance, see `after_newton_convergence` therein.

Reply:
My bad non-AD is not supported on this model. The assembler object was deleted and code was adapted accordingly.